### DISCUSSION TITLE
GCode for "Pause Print" in PrusaSlicer?

### DISCUSSION THREAD

**Discussion by prashnts** [mriscoc/Ender3V2S1#777] [Q&A] [answered]
Hi! I want to insert a nut mid-print without changing the filament, and with Print Head Parked somewhere. Specifically I want to be able to use "Pause at Height" feature in prusa.

In order to do that, I need to fill the "Pause Print G-Code" section in printer profile.

Based on other info here and elsewhere I found `M25`, but I haven't tried it yet because its description says it is `Pause SD Print`, while I'd be printing from OctoPrint. I'm also concern that it won't part the head.

I will try it out regardless, but I'd like to know the correct command to put there. Thanks

**Comment by mriscoc** [comment, marked as answer]
M25 is the correct command for a pause without filament change (M600)

**Reply by prashnts** [reply]
Thanks for the confirmation! Really enjoying the new firmware, kudos. :)
